OasisDEX/oasis-borrow
# open/manage earn heading

# [Earn Header Number](https://app.shortcut.com/oazo-apps/story/4478/simulate-earn-header-numbers)
According to Figma designs:
* New title of header in open/manage earn
* Value of yields and total value locked
  
## Changes 👷‍♀️
* New vault type: `earn`
* Earn Vault Type based on ilks (GUNI)
* New header for GUNI and manage view. 
  
## How to test 🧪
* New header should be displayed on the open earn view and manage view.
* Should work on connected and not connected context
